about.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="about.css">
+    <title>JSOCIETY Summit 2023- Conference Web Page</title>
+</head>
+<body>
+<!--HEADER-->
+    <div class="background">
+        <header class="header">
+            <div class="dark-nav">
+                <ul>
+                    <li><a href="https://www.facebook.com/"><img src="images/facebook.png" alt="fcebook icon"></a></li>
+                    <li><a href="https://twitter.com/home"><img src="images/twitter.png" alt="twitter icon"></a></li>
+                    <li><a href="/">English</a></li>
+                    <li><a href="/">My Page</a></li>
+                    <li><a href="/">Log out</a></li>
+                </ul>
```

Comment:
As per the previous reviewer's feedback! Regarding the position of the logout & community links, it's important to ensure that they are equidistant from the right edge. Please make the necessary adjustments to your code to achieve this alignment.

| Your | Design |
| --- | --- |
| ![image](https://github.com/peter-mwovi/Capstone-project/assets/55840999/21ea404c-b0d6-4886-bdd9-4e91be2320b2) | ![image](https://github.com/peter-mwovi/Capstone-project/assets/55840999/3454ff85-15c2-49c0-a65a-2261e38b3318) |


By properly aligning the logout & community links, you will enhance the visual appeal and consistency of your project. Keep up the good work! 👍🌟